app/src/androidTest/java/org/mozilla/focus/activity/DownloadFileTest.kt

```diff
@@ -135,4 +140,78 @@ class DownloadFileTest {
             assertNativeAppOpens(GOOGLE_PHOTOS)
         }
     }
+
+    @SmokeTest
+    @Test
+    fun downloadAndOpenPDFFileTest() {
+        downloadFileName = "washington.pdf"
+
+        searchScreen {
+        }.loadPage(downloadTestPage) {
+            progressBar.waitUntilGone(waitingTime)
+        }
+        browserScreen {
+            clickLinkMatchingText(downloadFileName)
```

Comment:
nit: Could this go in the same brackets as L151?

Reply:
Done!